webapp/mixins/PostMutationHelpers.js

```diff
@@ -0,0 +1,39 @@
+import gql from 'graphql-tag'
+
+export default {
+  methods: {
+    async deletePostCallback(postDisplayType = 'list') {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T13:02:34Z" title="Monday, June 3rd 2019, 3:02:34 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,39 @@
+import gql from 'graphql-tag'
+
+export default {
+  methods: {
+    async deletePostCallback(postDisplayType = 'list') {
```


As I said, try to avoid mixins. I guess this one might be replaced in a future refactoring. It's OK for now.